test/unit/UtilsTest.php

```diff
@@ -1,32 +1,29 @@
 <?php
+require_once 'test/helpers/EnvFactory.php';
+
 require_once 'src/wovnio/wovnphp/Utils.php';
 
+use Wovnio\Test\Helpers\EnvFactory;
+
 use Wovnio\Wovnphp\Store;
 use Wovnio\Wovnphp\Utils;
 
 class UtilsTest extends PHPUnit_Framework_TestCase {
-  private function getEnv($num="") {
-    $env = array();
-    $file = parse_ini_file(dirname(__FILE__) . '/mock_env' . $num . '.ini');
-    $env = $file['env'];
-    return $env;
-  }
-
   public function testFunctionsExists() {
     $this->assertTrue(class_exists('Wovnio\Wovnphp\Utils'));
     $this->assertTrue(method_exists('Wovnio\Wovnphp\Utils', 'getStoreAndHeaders'));
     $this->assertFalse(method_exists('Wovnio\Wovnphp\Utils', 'dispatchRequest'));
   }
 
   public function testGetStoreAndHeaders() {
-    $env = $this->getEnv('_path');
+    $env = EnvFactory::fromFixture('default');
```

Comment:
Because "default" is default argument for `fromFixture`, not needed?

Reply:
Yes, I actually wanted to remove the default because `EnvFactory::fromFixture()` does not make too much sens to me. I'll remove default value.

Reply:
I removed default value, thanks for review